.claude/prds/prd-skill-router-consolidation/phase-03-standalone-removal.md

```
@@ -0,0 +1,133 @@
+
+- [x] `git rm modules/shared/programs/claude/files/skills/prd/SKILL.md`.
+- [x] `git rm modules/shared/programs/claude/files/skills/prd/evals/queries.json`.
+- [x] `rmdir modules/shared/programs/claude/files/skills/prd/evals`.
+- [x] `rmdir modules/shared/programs/claude/files/skills/prd/references`.
+- [x] `rmdir modules/shared/programs/claude/files/skills/prd`.
+- [x] `git rm modules/shared/programs/claude/files/skills/review-implementation/SKILL.md`.
+- [x] `git rm modules/shared/programs/claude/files/skills/review-implementation/evals/queries.json`.
+- [x] `rmdir modules/shared/programs/claude/files/skills/review-implementation/evals`.
+- [x] `rmdir modules/shared/programs/claude/files/skills/review-implementation/references`.
+- [x] `rmdir modules/shared/programs/claude/files/skills/review-implementation`.
+- [x] `modules/shared/programs/claude/default.nix` Edit: line 234-240의 `# prd 스킬 ... ` + `.claude/skills/prd ...` + `# review-implementation 스킬` + `.claude/skills/review-implementation ...` declaration 모두 제거 (주석 포함).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)